### ISSUE TITLE
 'BleakClientCoreBluetooth' object has no attribute '_disconnected_callback_client'

### ISSUE THREAD

**Issue by dhalbert** [hbldh/bleak#294]
* bleak version: 57e35147a
* Operating System: MacOS 10.15.6

@dlech The disconnected callback code for MacOS seems to be missing setting an attribute. I tried one of my usual examples on `develop` immediately after #291 was merged, and got this error. I don't see this issue on bluez.
```
  ...
  File "/Users/halbert/repos/Adafruit_Blinka_bleio/_bleio/adapter_.py", line 359, in _connect_async
    await client.connect(timeout=timeout)
  File "/Users/halbert/repos/hbldh/bleak/bleak/backends/corebluetooth/client.py", line 88, in connect
    manager.disconnected_callback = self._disconnected_callback_client
AttributeError: 'BleakClientCoreBluetooth' object has no attribute '_disconnected_callback_client'
```

**Comment by dlech** [comment]
Sorry about that (careless mistake on my part). One fix coming right up.

**Comment by dhalbert** [comment]
Fixed by #295
